samples/drivers/adc/adc_sequence/boards/stm32f3_disco.overlay

```diff
@@ -0,0 +1,142 @@
+/ {
+	aliases {
+		adc0 = &adc_ad4114;
+	};
+};
+
+&spi1 {
+	pinctrl-0 = <&spi1_nss_pa4 &spi1_sck_pa5
+		     &spi1_miso_pa6 &spi1_mosi_pa7>;
+	pinctrl-names = "default";
+	status = "okay";
+	label = "spi1";
+
+	adc_ad4114: adc_ad4114@0 {
+		compatible = "adi,ad4114-adc";
+		spi-max-frequency = <1000000>;
+		status = "okay";
+		reg = <0>; /* ignored, but necessary for SPI bindings */
```

Comment:
This isn't ignored; it's the chip select index

Reply:
You are right, comment removed